src/Sulu/Bundle/AdminBundle/Metadata/ListMetadata/ListMetadata.php

```diff
@@ -37,4 +38,18 @@ public function addField(FieldMetadata $field): void
     {
         $this->fields[$field->getName()] = $field;
     }
+
+    public function getField(string $fieldName): FieldMetadata
+    {
+        if (!isset($this->fields[$fieldName])) {
+            throw new FieldMetadataNotFoundException($fieldName);
```

Comment:
sadly the exception can not show which listMetadata the field is missing as the listMetadata doesn't now its key.